static/src/content/doc/getting-started.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started
+description: Getting started with Batteries Included.
+tags: ['getting-started', 'overview']
+category: getting-started
+draft: false
+---
+
+# Getting Started with Batteries Included
+
+Whether you're building web services, running AI workloads, or managing
+databases, getting started takes just a few minutes:
+
+1. **Choose Where to Run**: Deploy locally for development or to your preferred
+   cloud environment.
+2. **Tell Us What You Need**: Select the components you want to run - databases,
+   web services, AI tools, monitoring, etc.
```

Comment:
I like this structure with no wind up. However I think each of the steps will likely still need more verbiage. For example in the last version you had a description about kind, AWS, or already running kubernetes cluster.  That was helpful I thought. Also a cut down screenshot (less than full screen, we can cut the sides and top off of the new install screen) showing how to choose where, and what to run would be really helpful.